### DISCUSSION TITLE
Does this firmware support flysky small multiprotocol module?

### DISCUSSION THREAD

**Discussion by jaytxrx** [OpenI6X/opentx#335] [Q&A] [answered]
Does this opentx for i6x support the tiny multiprotocol module ?

https://aliexpress.com/item/1005003810274204.html

![image](https://user-images.githubusercontent.com/431506/211173751-1cb9c2f5-084b-41c7-a656-58ad27c0612a.png)

**Comment by Ultraexpert** [comment]
I think in the past versions this would not work. I hope this issue is now resolved!

**Comment by ajjjjjjjj** [comment, marked as answer]
If PPM Out now works for you then this module should also. I don't have it to test it by myself.

**Comment by jaytxrx** [comment]
Is it also possible to connect ExpressLRS module on this port ?
I see that wire is soldered to the PCB to connect expressLRS modules and not to this port. Any reasons for that ?

**Reply by ajjjjjjjj** [reply]
Yes, it's on different pad for a reason - hardware serial data output with build in inversion. Any other radio also don't have it on the same pin.

**Reply by jaytxrx** [reply]
Sorry. I didn't understand. 

Do you mean that serial data on the trainer port is inverted and Elrs cannot work with that ? And hence we use the internal PCB to connect elrs ?

**Reply by ajjjjjjjj** [reply]
No, CRSF needs serial with inversion.

**Reply by jaytxrx** [reply]
Is it possible to convert the PPM signal to serial with inversion and use that for expressLrs ? I mean in software map the serial pin to the ppm pin. In that case we can connect expresslrs module to the trainer port.

**Reply by ajjjjjjjj** [reply]
No, it's not possible.

**Reply by jaytxrx** [reply]
Why is it so ? Is it a hardware issue ? Can you please elaborate ?

**Reply by ajjjjjjjj** [reply]
It's impossible to configure MCU in i6X to work like that.
